components/clp-py-utils/clp_py_utils/s3_utils.py

```diff
@@ -0,0 +1,42 @@
+from pathlib import Path
+
+import boto3
+from botocore.config import Config
+from botocore.exceptions import ClientError
+
+from clp_py_utils.clp_config import S3Config
+from clp_py_utils.result import Result
+
+
+def s3_put(
+    s3_config: S3Config, src_file: Path, dest_file_name: str, total_max_attempts: int = 3
+) -> Result:
+
+    if not src_file.exists():
+        return Result(success=False, error=f"{src_file} doesn't exist")
+    if not src_file.is_file():
+        return Result(success=False, error=f"{src_file} is not a file")
+
+    s3_client_args = {
+        "region_name": s3_config.region_name,
```

Comment:
Shoudn't `region_name` be in the config object? https://botocore.amazonaws.com/v1/documentation/api/latest/reference/config.html#botocore-config

Reply:
Can we resolve this? @kirkrodrigues

Reply:
Yep